src/store.ts

```diff
@@ -260,11 +267,18 @@ export class DynamoDBStore extends session.Store {
           S: sid,
         }
       },
-      UpdateExpression: 'set expires = :e',
+      UpdateExpression: 'set expires = :e, #ttl = :ttl',
+      ExpressionAttributeNames: {
+        '#ttl': this.options.ttlFieldName
+      },
       ExpressionAttributeValues: {
         ':e': {
           N: JSON.stringify(expires),
         },
+        ':ttl': {
+          N: JSON.stringify(this.getTTLValue())
```

Comment:
It says this is the fastest way to convert number to a string value. It's a bit odd to use JSON.stringify here, it's slower and confusing because getTTLValue returns a number.
```suggestion
          N: '' + this.getTTLValue()
```